assets/src/dashboard/app/views/shared/templateGridView.js

```diff
@@ -62,32 +72,91 @@ function TemplateGridView({ pageSize, templates, templateActions }) {
     [templateActions]
   );
 
+  useGridViewKeys({
+    containerRef,
+    gridRef,
+    itemRefs,
+    isRTL,
+    currentItemId: activeGridItemId,
+    items: templates,
+  });
+
+  // when keyboard focus changes through FocusableGridItem immediately focus the CardPreviewContainer
+  useEffect(() => {
+    itemRefs.current?.[activeGridItemId]?.lastChild.focus();
+  }, [activeGridItemId]);
+
   return (
-    <GridContainer pageSize={pageSize}>
-      {templates.map((template) => (
-        <CardGridItem
-          key={template.id}
-          data-testid={`template-grid-item-${template.id}`}
-        >
-          <CardPreviewContainer
-            pageSize={pageSize}
-            story={template}
-            centerAction={{
-              targetAction: template.centerTargetAction,
-              label:
-                TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS[template.status],
-            }}
-            bottomAction={{
-              targetAction: targetAction(template),
-              label: __('Use template', 'web-stories'),
-            }}
-            containerAction={(e) =>
-              templateActions.handlePreviewTemplate(e, template)
-            }
-          />
-        </CardGridItem>
-      ))}
-    </GridContainer>
+    <div ref={containerRef}>
+      <GridContainer
+        pageSize={pageSize}
+        gridRef={gridRef}
+        ariaLabel={__('Available templates', 'web-stories')}
+      >
+        {templates.map((template) => {
+          const isActive = activeGridItemId === template.id;
+          const tabIndex = isActive ? 0 : -1;
+          return (
+            <CardGridItem
+              key={template.id}
+              data-testid={`template-grid-item-${template.id}`}
```

Comment:
Let's pass an aria role here because right now it's speaking "Available templates, list with 0 items"

Reply:
that's odd because it is a `listItem` (components/cardGridItem/index)
`styled.div.attrs({ role: 'listItem' })` - i'll dig into this in the am

Reply:
the updated labeling should resolve this.